### ISSUE TITLE
Artist details

### ISSUE THREAD

**Issue by richardofsussex** [national-gallery/NG-CIIM#9]
I have implemented a minimal pattern for artist within the produced_by section:

![image](https://user-images.githubusercontent.com/2197102/231426877-3ead1d79-ff83-425d-9087-cc9714d71eb0.png)

Is this sufficient, or do you want me to include the biographical details which appear in the source JSON (date of birth and death)? Strictly speaking, users can get this information by dereferencing the URI for Titian. [You'll notice I have cheated in relation to the entity type issue!]

**Comment by RGShepherd** [comment]
> Just a comment: the type value is probably used to resolve against the context document, e.g. type: "Person" asserts that this carried_out_by (crm:P14_carried_out_by) 'predicate' has as its 'object' a Person (crm:E21_Person). If we don't assume this, there is no way to resolve that structure against the context document. If that's the case, it _is_ essential.

But note that this cannot be inferred from the object record in the CIIM - it does require us to extract it from the relevant agent record upon mapping.

**Comment by richardofsussex** [comment]
OK: I'll explore whether I can pull in the agent data using the document() function. (This is normally used to fetch XML resources, so it's not a given that it will be happy to fetch a JSON document.)

**Comment by richardofsussex** [comment]
Yes: success. I now pull in the NG @datatype.actual for the creator, and if it's 'Individual' I output the LA type 'Person'; otherwise I output the LA type 'Group'.
